test/benchmark/bm_iroha_ed25519.cpp

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ed25519/ed25519.h>
+
+#include <cstdlib>
+#include <vector>
+
+#include <benchmark/benchmark.h>
+
+auto ConstructRandomVector(int64_t size) {
+  std::vector<unsigned char> v;
+  v.reserve(static_cast<int>(size));
+  for (int i = 0; i < size; ++i) {
+    v.push_back(static_cast<unsigned char>(std::rand() % size));
+  }
+  return v;
+}
+
+static void BM_CreateKeypair(benchmark::State &state) {
+  public_key_t pub{};
```

Comment:
Looks like you do not need {} here and below.

Reply:
Unfortunately they are needed to initialize the struct which does not have a constructor.